packages/@aws-cdk/aws-secretsmanager/lib/secret.ts

```diff
@@ -297,6 +335,35 @@ export class Secret extends SecretBase {
     }(scope, id);
   }
 
+  /**
+   * Imports a secret by secret name.
+   * A secret with this name must exist in the same account & region.
+   * Replaces the deprecated `fromSecretName`.
+   */
+  public static fromSecretNameV2(scope: Construct, id: string, secretName: string): ISecret {
```

Comment:
I know the names are tough here, but I don't love `fromSecretNameV2`. How about something like `fromSecretNameWithoutSuffix` / `fromSecretNameNoSuffix`?

Reply:
Discussed that briefly (with Elad) but there is no reasonable use case for `fromSecretName**With**Suffix`, and `fromSecretNameWithoutSuffix` seems to imply that `fromSecretName` _should_ be used if you have the suffix. Neither name is good, but I feel just calling it 'V2' makes it clear it's an improved version of the former, rather than a version skewed toward a particular use case.